features/admin.branding.v1/context/branding-preference-context.tsx

```diff
@@ -39,6 +40,15 @@ export interface BrandingPreferenceContextProps {
      * @param screen - Selected screen.
      */
     onSelectedPreviewScreenChange: (screen: PreviewScreenType) => void;
+    /**
+     * Sets the variation of the current screen, requested for preview.
+     * @param variation - Selected screen variation.
+     */
+    onSelectedPreviewScreenVariationChange: (variation: PreviewScreenVariationType) => void;
+    /**
+     * Resets the variations back to default
```

Comment:
full stop at the end

Reply:
addressed with https://github.com/wso2/identity-apps/pull/6027/commits/6f514ee32c24be59d56b0f82cd4c13abfa87c27b